docs/advanced_features.md

```diff
@@ -0,0 +1,71 @@
+# Advanced features
+
+## SPARQL+Text and SPARQL Autocompletion
+
+On top of the vanilla SPARQL functionality, QLever allows so-called SPARQL+Text
+queries on a text corpus linked to a knowledge base via entity recognition.  For
+example, the following query find all mentions of astronauts next to the words
+"moon" and "walk*" in the text corpus:
+
+    SELECT ?a TEXT(?t) SCORE(?t) WHERE {
+        ?a <is-a> <Astronaut> .
+        ?t ql:contains-entity ?a .
+        ?t ql:contains-word "walk* moon"
+    } ORDER BY DESC(SCORE(?t))
+
+Such queries can be simulated in standard SPARQL, but only with poor
+performance, see the CIKM'17 paper above.  Details about the required input data
+and the SPARQL+text query syntax and semantics can be found
+[here](docs/sparql_plus_text.md).
+
+QLever also supports efficient SPARQL autocompletion.  For example, the
+following query yields a list of all predicates associated with persons in the
+knowledge base, ordered by the number of persons which have that predicate.
```

Comment:
`pesons` -> `people`

Reply:
Interestingly it turns out "persons" is [okay too](https://www.dailywritingtips.com/people-versus-persons/) and is in fact the more formal (older) form. Still, since we are using C++17 we might as well stick to the more modern form for English as well. So thanks for pointing this out to me!